Formula/doctl.rb

```diff
@@ -3,8 +3,8 @@
 class Doctl < Formula
   desc "Command-line tool for DigitalOcean"
   homepage "https://github.com/digitalocean/doctl"
-  url "https://github.com/digitalocean/doctl/archive/v1.0.2.tar.gz"
-  sha256 "7a28bd0f9e68eee41761e32267a0d63329c6f0e16e0d8b5afb4a8286b914e98a"
+  url "https://github.com/digitalocean/doctl/archive/v1.1.0.tar.gz"
```

Comment:
Yup, sorry about that. I'll reopen when that's solved to avoid wasting more of you time. Apologies!